docs/tutorial.md

```diff
@@ -128,31 +128,26 @@ working copy commit by default.
 ## The log command, "revsets", and aliases
 
 You're probably familiar with `git log`. Jujutsu has very similar functionality
-in its `jj log` command. It produces hundreds of lines of output, so let's pipe
-its output into `head`:
+in its `jj log` command:
 ```shell script
-$ jj log | head
+$ jj log
 @ 192b456b024b f39aeb1a0200 martinvonz@google.com 2021-05-23 23:10:27.000 -07:00
 |
 o fb563a4c6d26 f63e76f175b9 martinvonz@google.com 2021-05-23 22:13:45.000 -07:00
 | Jujutsu is ready!
 o 080a9b37ff7e 6a91b4ba16c7 martinvonz@google.com 2021-05-23 22:08:37.000 -07:00 main
-| cli: make `jj st` show parent commit before working copy commit
-o ba8ff31e32fd 302257bdb7e5 martinvonz@google.com 2021-05-23 22:08:12.000 -07:00
-| cli: make the working copy changes in `jj status` clearer
-o dcfc888f50b3 7eddf8dfc70d martinvonz@google.com 2021-05-23 22:07:40.000 -07:00
-| cli: remove "Done" message at end of git clone
+~ cli: make `jj st` show parent commit before working copy commit
```

Comment:
It might be worth explaining what `~` means in the log output.

Reply:
Good point. Done.